news/2022-05-19-osu-world-cup-2022-mapper-tester-applications.md

```diff
@@ -0,0 +1,31 @@
+---
+layout: post
+title: "osu! World Cup 2022: Mapper & Tester Applications"
+date: 2022-05-19 09:00:00 +0000
+---
+
+The 2022 osu! World Cup is slated to the latter half of the year, but preparations are already underway. Read on if you are intested in helping out with mappooling for the year's biggest tournament!
+
+![](/wiki/shared/news/banners/OWC_Master_Logo.png)
+
+The OWC mappooling team has changed a lot this year and we are always in need of some additional hands in order to continue creating a fantastic experience for players and viewers alike. This year the team so far includes 10 mappoolers with most able to fulfil the high quality checking requirement that a major tournament like OWC demands, and we are looking to add around 10 more playtesters to make sure everything is as best it can be. There will be 50 to 60 custom made maps included in the tournament, so there is a great opportunity for many different types of mappers to join the team!
```

Comment:
```suggestion
The OWC mappooling team has changed a lot this year and we are always in need of some additional hands in order to continue creating a fantastic experience for players and viewers alike. This year the team so far includes 10 mappoolers with most able to fulfil the high-quality checking requirement that a major tournament like OWC demands, and we are looking to add around 10 more playtesters to make sure everything is as best it can be. There will be 50 to 60 custom-made maps included in the tournament, so there is a great opportunity for many different types of mappers to join the team!
```